src/Bicep.Cli/Program.cs

```diff
@@ -40,11 +40,22 @@ public static Task<int> Main(string[] args)
 
             BicepDeploymentsInterop.Initialize();
 
+            if (bool.TryParse(Environment.GetEnvironmentVariable("BICEP_TRACING_ENABLED"), out var enableTracing) && enableTracing)
+            {
+                Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
+            }
+
+            var settings = new EmitterSettings(ThisAssembly.AssemblyFileVersion, enableSymbolicNames: false);
+            if (bool.TryParse(Environment.GetEnvironmentVariable("BICEP_SYMBOLIC_NAME_CODEGEN_EXPERIMENTAL"), out var enableSymbolicNames) && enableSymbolicNames)
```

Comment:
>BICEP_SYMBOLIC_NAME_CODEGEN_EXPERIMENTAL [](http://example.com/codeflow?start=66&length=40)

Shouldn't we do this through `IFeaturesProvider`?

Reply:
Yeah, let me give that a go. This PR was created before `IFeaturesProvider` existed.

Reply:
Done!